health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/editVillagePopulationPopUP.js

```
@@ -76,72 +79,81 @@ const EditVillagePopulationPopUp = ({ onClose, census, onSuccess }) => {
 
   const handleInputChange = (fieldKey, value) => {
-    setTargetExceed(false);
     setConfirmedValues({
       ...confirmedValues,
       [fieldKey]: value,
     });
   
-    let error = !value || Number(value) <= 0 || Number(value) > 100000 || !Number.isInteger(Number(value));
-    if(error===true){
-      setErrorMessage({ key: "error", label: t("HCM_MICROPLAN_ONLY_POSITIVE_NUMBERS_MAX_LIMIT_VALIDATION_ERROR") });
+    let newErrors = { ...errors }; 
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'newErrors' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)